src/mlpack/tests/cv_test.cpp

```diff
@@ -856,3 +857,177 @@ TEST_CASE("SilhouetteScoreTest", "[CVTest]")
   double silhouetteScore = SilhouetteScore::Overall(X, labels, metric);
   REQUIRE(silhouetteScore == Approx(0.1121684822489150).epsilon(1e-7));
 }
+
+class RandomForestFacade : public RandomForest<>
+{
+ public:
+  using RandomForest<>::RandomForest;
+
+  void Train(
+      const arma::mat& data,
+      const arma::Row<size_t>& labels,
+      size_t numClasses)
+  {
+    [[maybe_unused]] const double _(static_cast<RandomForest<>*>(this)->Train(
+        data,
+        labels,
+        numClasses));
+  }
+};
```

Comment:
Are you sure this class is necessary?  `RandomForest` should already work with k-fold cross-validation as it is (it should have the right form of `Train()` overload).

Reply:
My perspective is the following: `RandomForest`'s `Train` method has a `double` return type, while `MetaInfoExtractor` requires `void` return type.